packages/react-progress-bar/lib/react-progress-bar.js

```diff
@@ -14,8 +14,13 @@ export const ProgressBar = ({
 	...props
 }) => {
 	const extraClasses = 'undefined' !== typeof classes && '' !== classes ? ' ' + classes : '';
-	const value = 'undefined' !== typeof now && !Number.isNaN(now) ? now : 0;
-
+	// check whether value is valid or not.
+	const getValue = value => {
+		if (value === undefined || Number.isNaN(value) || value === null || value.length === 0)
```

Comment:
@creador-dev, a couple of things to improve:

- [ ] Use `yoda` conditions.
- [ ] Remember to use the `typeof` operator on conditions so you can check the type of variable you’re getting.